lib/matplotlib/animation.py

```diff
@@ -432,9 +456,16 @@ def finish(self):
         # Check error code for creating file here, since we just run
         # the process here, rather than having an open pipe.
         if self._proc.returncode:
+            try:
+                stdout = [s.decode() for s in self._proc._stdout_buff]
+                stderr = [s.decode() for s in self._proc._stderr_buff]
+                verbose.report("MovieWriter.finish: stdout: %s" % stdout, level='helpful')
+                verbose.report("MovieWriter.finish: stderr: %s" % stderr, level='helpful')
+            except Exception as e:
+                pass
```

Comment:
yes, when the called converter fails and the user adds the `mpl.verbose.set_level("helpful")` before calling the failing method again...

Unfortunately, it doesn't work for the unittests unless I add such a `set_level` call in the unittest file :-(